### PR TITLE
Proposal executor

### PR DESCRIPTION
**Pull request by true-eye**
## Description

image

image

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ ] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [ ] no

### CODE REVIEW

File: client/scripts/views/pages/new_proposal/new_proposal_form.ts [hunk 1]
```diff
@@ -980,15 +977,11 @@ const NewProposalForm = {
             ]),
             m(FormGroup, [
               m(FormLabel, 'Proposal Executor'),
-              m(DropdownFormField, {
-                choices: (app.chain as Aave).governance.api.Executors.map(
-                  (r) => ({ name: 'executor', value: r.address, label: `${r.delay / (60 * 60 * 24)} Day(s) Voting` })
-                ),
-                callback: (result) => {
-                  vnode.state.executor = result;
```
Comment (true-eye) [inline client/scripts/views/pages/new_proposal/new_proposal_form.ts:988]: @jnaviask I am not sure why the two different dropdowns mutates the same state `vnode.state.executor`

Comment (zakhap) [reply client/scripts/views/pages/new_proposal/new_proposal_form.ts:988]: Because they refer to the same thing!

**Comment by jnaviask** [comment]
This display doesn't align with the figma spec -- we need to display delay time for each executor as well as address, and we may ultimately need more visual information about each one.

**Comment by true-eye** [comment]
> This display doesn't align with the figma spec -- we need to display delay time for each executor as well as address, and we may ultimately need more visual information about each one.

Can you share me the figma link, please?

**Comment by RakeshUP** [comment]
Hey @true-eye, I too started working on this issue and am nearing completion. There's some work pending with the tabs(Call 1, 2) section. Screenshot of my current progress. Letting you know so that we both don't work on the same stuff..

Screenshot 2021-08-02 at 11 20 26 PM

**Comment by true-eye** [comment]
> Hey @true-eye, I too started working on this issue and am nearing completion. There's some work pending with the tabs(Call 1, 2) section. Screenshot of my current progress. Letting you know so that we both don't work on the same stuff..
> 
> Screenshot 2021-08-02 at 11 20 26 PM

oops. I didn't know you were in progress. I have been working on this but you can continue on it. I will move onto another. Thanks for letting me know @RakeshUP

**Pull request closed without merging**